### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/tutorial/env.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#579] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/tutorial/env.html

本教程将指导你如何搭建 VuePress 的运行环境。

https://theme-hope.vuejs.press/zh/cookbook/tutorial/env.html

**Comment by huamus** [comment]
安装pnmp时如果报错，可以试试以管理员身份运行cmd
